### PR TITLE
[2/2] DXCDT-129: Add vcr recordings for tests

### PR DESCRIPTION
**Pull request by sergiughf**
## Description

This PR adds the recordings for our go-vcr tests. 

Note: It does not yet enable the CI tests to use them. This will be done in the next PR. 

## Testing

- [x] This change adds test coverage for new/changed/fixed functionality

## Checklist

- [x] I have read and agreed to the terms within the [Auth0 Code of Conduct](https://github.com/auth0/open-source-template/blob/master/CODE-OF-CONDUCT.md).
- [x] I have read the [Auth0 General Contribution Guidelines](https://github.com/auth0/open-source-template/blob/master/GENERAL-CONTRIBUTING.md).
- [x] I have reviewed my own code beforehand.
- [x] I have added documentation for new/changed functionality in this PR.
- [x] All active GitHub checks for tests, formatting, and security are passing.
- [x] The correct base branch is being used, if not `main`.

### CODE REVIEW

**Review by evansims — APPROVED** [review]
LGTM; is the failing test expected at this point?

**Review by willvedd — APPROVED** [review]
Thanks for breaking this out into it's own PR. Good work!

File: management/testdata/recordings/TestConnectionManager_Create/It_can_successfully_create_a_SAML_Connection.yaml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
...GA1UEAxMaZXVyb3Bh\nLnNmby5jb3JwLmdvb2dsZS5jb20wggIiMA0GCSqGSIb3DQEBAQUAA4ICDwAwggIK\nAoICAQC6pgYt7/EibBDumASF+S0qvqdL/f+nouJw2T1Qc8GmXF/iiUcrsgzh/Fd8\npDhz/T96Qg9IyR4ztuc2MXrmPra+zAuSf5bevFReSqvpIt8Duv0HbDbcqs/XKPfB\nuMDe+of7a9GCywvAZ4ZUJcp0thqD9fKTTjUWOBzHY1uNE4RitrhmJCrbBGXbJ249\nbvgmb7jgdInH2PU7PT55hujvOoIsQW2osXBFRur4pF1wmVh4W4lTLD6pjfIMUcML\nICHEXEN73PDic8KS3EtNYCwoIld+tpIBjE1QOb1KOyuJBNW6Esw9ALZn7stWdYcE\nqAwvv20egN2tEXqj7Q4/1ccyPZc3PQgC3FJ8Be2mtllM+80qf4dAaQ/fWvCtOrQ5\npnfe9juQvCo8Y0VGlFcrSys/MzSg9LJ/24jZVgzQved/Qupsp89wVidwIzjt+WdS\nfyWfH0/v1aQLvu5cMYuW//C0W2nlYziL5blETntM8My2ybNARy3ICHxCBv2RNtPI\nWQVm+E9/W5rwh2IJR4DHn2LHwUVmT/hHNTdBLl5Uhwr4Wc7JhE7AVqb14pVNz1lr\n5jxsp//ncIwftb7mZQ3DF03Yna+jJhpzx8CQoeLT6aQCHyzmH68MrHHT4MALPyUs\nPomjn71GNTtDeWAXibjCgdL6iHACCF6Htbl0zGlG0OAK+bdn0QIDAQABMA0GCSqG\nSIb3DQEBBQUAA4GBAOKnQDtqBV24vVqvesL5dnmyFpFPXBn3WdFfwD6DzEb21UVG\n5krmJiu+ViipORJPGMkgoL6BjU21XI95VQbun5P8vvg8Z+FnFsvRFY3e1CCzAVQY\nZsUkLw2I7zI/dNlWdB8Xp7v+3w9sX5N3J/WuJ1KOO5m26kRlHQo7EzT3974g\n-----END CERTIFICATE-----","tenant_domain":"example.com","signInEndpoint":"https://saml.identity/provider","fieldsMap":{"email":"EmailAddress","family_name":"LastName","given_name":"FirstName"}}}
```
Comment (Widcket) [inline management/testdata/recordings/TestConnectionManager_Create/It_can_successfully_create_a_SAML_Connection.yaml:6]: Should this be redacted as well?

File: management/testdata/recordings/TestConnectionManager_Read/It_can_successfully_read_a_SAML_Connection.yaml [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
...GA1UEAxMaZXVyb3Bh\nLnNmby5jb3JwLmdvb2dsZS5jb20wggIiMA0GCSqGSIb3DQEBAQUAA4ICDwAwggIK\nAoICAQC6pgYt7/EibBDumASF+S0qvqdL/f+nouJw2T1Qc8GmXF/iiUcrsgzh/Fd8\npDhz/T96Qg9IyR4ztuc2MXrmPra+zAuSf5bevFReSqvpIt8Duv0HbDbcqs/XKPfB\nuMDe+of7a9GCywvAZ4ZUJcp0thqD9fKTTjUWOBzHY1uNE4RitrhmJCrbBGXbJ249\nbvgmb7jgdInH2PU7PT55hujvOoIsQW2osXBFRur4pF1wmVh4W4lTLD6pjfIMUcML\nICHEXEN73PDic8KS3EtNYCwoIld+tpIBjE1QOb1KOyuJBNW6Esw9ALZn7stWdYcE\nqAwvv20egN2tEXqj7Q4/1ccyPZc3PQgC3FJ8Be2mtllM+80qf4dAaQ/fWvCtOrQ5\npnfe9juQvCo8Y0VGlFcrSys/MzSg9LJ/24jZVgzQved/Qupsp89wVidwIzjt+WdS\nfyWfH0/v1aQLvu5cMYuW//C0W2nlYziL5blETntM8My2ybNARy3ICHxCBv2RNtPI\nWQVm+E9/W5rwh2IJR4DHn2LHwUVmT/hHNTdBLl5Uhwr4Wc7JhE7AVqb14pVNz1lr\n5jxsp//ncIwftb7mZQ3DF03Yna+jJhpzx8CQoeLT6aQCHyzmH68MrHHT4MALPyUs\nPomjn71GNTtDeWAXibjCgdL6iHACCF6Htbl0zGlG0OAK+bdn0QIDAQABMA0GCSqG\nSIb3DQEBBQUAA4GBAOKnQDtqBV24vVqvesL5dnmyFpFPXBn3WdFfwD6DzEb21UVG\n5krmJiu+ViipORJPGMkgoL6BjU21XI95VQbun5P8vvg8Z+FnFsvRFY3e1CCzAVQY\nZsUkLw2I7zI/dNlWdB8Xp7v+3w9sX5N3J/WuJ1KOO5m26kRlHQo7EzT3974g\n-----END CERTIFICATE-----","tenant_domain":"example.com","signInEndpoint":"https://saml.identity/provider","fieldsMap":{"email":"EmailAddress","family_name":"LastName","given_name":"FirstName"}}}
```
Comment (Widcket) [inline management/testdata/recordings/TestConnectionManager_Read/It_can_successfully_read_a_SAML_Connection.yaml:6]: Should this be redacted as well?

File: management/testdata/recordings/TestConnectionManager_Read/It_can_successfully_read_a_SAML_Connection.yaml [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
...AbBgkq\nhkiG9w0BCQEWDmFnbEBnb29nbGUuY29tMB4XDTA5MDkwOTIyMDU0M1oXDTEwMDkw\nOTIyMDU0M1owajELMAkGA1UEBhMCQVUxEzARBgNVBAgTClNvbWUtU3RhdGUxITAf\nBgNVBAoTGEludGVybmV0IFdpZGdpdHMgUHR5IEx0ZDEjMCEGA1UEAxMaZXVyb3Bh\nLnNmby5jb3JwLmdvb2dsZS5jb20wggIiMA0GCSqGSIb3DQEBAQUAA4ICDwAwggIK\nAoICAQC6pgYt7/EibBDumASF+S0qvqdL/f+nouJw2T1Qc8GmXF/iiUcrsgzh/Fd8\npDhz/T96Qg9IyR4ztuc2MXrmPra+zAuSf5bevFReSqvpIt8Duv0HbDbcqs/XKPfB\nuMDe+of7a9GCywvAZ4ZUJcp0thqD9fKTTjUWOBzHY1uNE4RitrhmJCrbBGXbJ249\nbvgmb7jgdInH2PU7PT55hujvOoIsQW2osXBFRur4pF1wmVh4W4lTLD6pjfIMUcML\nICHEXEN73PDic8KS3EtNYCwoIld+tpIBjE1QOb1KOyuJBNW6Esw9ALZn7stWdYcE\nqAwvv20egN2tEXqj7Q4/1ccyPZc3PQgC3FJ8Be2mtllM+80qf4dAaQ/fWvCtOrQ5\npnfe9juQvCo8Y0VGlFcrSys/MzSg9LJ/24jZVgzQved/Qupsp89wVidwIzjt+WdS\nfyWfH0/v1aQLvu5cMYuW//C0W2nlYziL5blETntM8My2ybNARy3ICHxCBv2RNtPI\nWQVm+E9/W5rwh2IJR4DHn2LHwUVmT/hHNTdBLl5Uhwr4Wc7JhE7AVqb14pVNz1lr\n5jxsp//ncIwftb7mZQ3DF03Yna+jJhpzx8CQoeLT6aQCHyzmH68MrHHT4MALPyUs\nPomjn71GNTtDeWAXibjCgdL6iHACCF6Htbl0zGlG0OAK+bdn0QIDAQABMA0GCSqG\nSIb3DQEBBQUAA4GBAOKnQDtqBV24vVqvesL5dnmyFpFPXBn3WdFfwD6DzEb21UVG\n5krmJiu+ViipORJPGMkgoL6BjU21XI95VQbun5P8vvg8Z+FnFsvRFY3e1CCzAVQY\nZsUkLw2I7zI/dNlWdB8Xp7v+3w9sX5N3J/WuJ1KOO5m26kRlHQo7EzT3974g\n-----END
```
Comment (Widcket) [inline management/testdata/recordings/TestConnectionManager_Read/It_can_successfully_read_a_SAML_Connection.yaml:16]: Same as above.

File: management/testdata/recordings/TestConnectionManager_Read/It_can_successfully_read_a_SAML_Connection.yaml [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
```
Comment (Widcket) [inline management/testdata/recordings/TestConnectionManager_Read/It_can_successfully_read_a_SAML_Connection.yaml:19]: Same as above.

File: management/testdata/recordings/TestConnectionManager_Read/It_can_successfully_read_a_SAML_Connection.yaml [hunk 5]
```diff
@@ -0,0 +1,85 @@
...
```
Comment (Widcket) [inline management/testdata/recordings/TestConnectionManager_Read/It_can_successfully_read_a_SAML_Connection.yaml:39]: Same as above.

File: management/testdata/recordings/TestConnectionManager_Read/It_can_successfully_read_a_SAML_Connection.yaml [hunk 6]
```diff
@@ -0,0 +1,85 @@
...
```
Comment (Widcket) [inline management/testdata/recordings/TestConnectionManager_Read/It_can_successfully_read_a_SAML_Connection.yaml:42]: Same as above.

**Review by Widcket — CHANGES_REQUESTED** [review]
Left some comments about what appears to be some certs, not sure if it's relevant.

**Review by Widcket — APPROVED** [review]

**Pull request merged**